Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -469,6 +480,132 @@ define([
          */
         this.freezeFrame = false;
 
+        var onlyPickedTileInfo = knockout.observable();
+        knockout.defineProperty(this, 'onlyPickedTileInfo', {
+            get : function() {
+                return onlyPickedTileInfo();
+            },
+            set : function(val) {
+                onlyPickedTileInfo(val);
+                if (that._tileset) {
+                    that._tileset.debugShowOnlyPickedTile = val;
+                }
+            }
+        });
```

Comment:
As mentioned in my other comment, the per-tile picking could be saved for another PR.